joomla/joomla-cms
# [4.0] Alternative menu

PR for #27737

### Summary

- Add missing language strings
- Change resources menu link
- Ensure that all utems have the correct url

### Testing Before

- Create a new admin menu with the alternative preset and name it **test1**
- View the list of created menu items
- Observe several missing language strings
- Open the Global Checkin, Clear Cache or System Information menu item
- Observe that Menu Item Type is blank

### Testing After Apply PR

- Create a new admin menu with the alternative preset and name it test2
- View the list of created menu items
- Observe no missing language strings
- Open the Global Checkin, Clear Cache or System Information menu item
- Observe that Menu Item Type is completed


### Before
![image](https://user-images.githubusercontent.com/1296369/114263354-e0e6a980-99dc-11eb-9864-1d916ff5f695.png)

![image](https://user-images.githubusercontent.com/1296369/114263362-f360e300-99dc-11eb-8990-fbca0777cd8c.png)
